sim/hunter/item_sets_pvp.go

```diff
@@ -128,3 +128,99 @@ var ItemSetLieutenantCommandersProwess = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+///////////////////////////////////////////////////////////////////////////
+//                            SoD Phase 5 Item Sets
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetWarlordsPursuit = core.NewItemSet(core.ItemSet{
+	Name: "Warlord's Pursuit",
+	Bonuses: map[int32]core.ApplyEffect{
+		// +20 Agility, 20 Stamina
+		2: func(agent core.Agent) {
+			c := agent.GetCharacter()
+			c.AddStats(stats.Stats{
+				stats.Agility:       20,
+				stats.Stamina:       20,
+			})
+		},
+		// Reduces the cooldown of your Concussive Shot by 1 sec.
+		4: func(agent core.Agent) {
+			// Nothing to do
+		},
+		// +20 Agi, 23 Spellpower.
+		6: func(agent core.Agent) {
+			c := agent.GetCharacter()
+			c.AddStats(stats.Stats{
+				stats.Agility:           20,
+				stats.Spellpower:        23,
+			})
+		},
+	},
+})
+
+var ItemSetWarlordsProwess = core.NewItemSet(core.ItemSet{
```

Comment:
The Prowess sets also have a +4% raptor strike damage mod on the gloves if you wouldn't mind adding those in `sim/hunter/items.go` too
<img width="403" alt="image" src="https://github.com/user-attachments/assets/ebd1392d-fc4a-4741-bca0-0c58a9a5ccc4">

Reply:
I believe both this pair and the other pair are included in this pull from Docoda:
https://github.com/wowsims/sod/pull/1001

Reply:
That's totally fine then! I know @ncberman is taking a look at Docoda's PR to get it cleaned up and passing so we'll get them there!